src/main/java/bio/terra/cli/command/cromwell/GenerateConfig.java

```diff
@@ -49,6 +70,18 @@ protected void execute() {
     String googleProjectId = Context.requireWorkspace().getRequiredGoogleProjectId();
     String petSaEmail = Context.requireUser().getPetSaEmail();
 
+    String googleBucket;
+    if (workspace_or_google_bucket_name.workspace_bucket_name == null) {
+      googleBucket = workspace_or_google_bucket_name.google_bucket_name;
+      if (!googleBucket.startsWith("gs://")) googleBucket = "gs://" + googleBucket;
+    } else {
+      Resource resource =
+          Context.requireWorkspace()
+              .getResource(workspace_or_google_bucket_name.workspace_bucket_name);
+      boolean includeUrlPrefix = true;
```

Comment:
nit: inline the boolean

Reply:
Sorry, I'm not sure what you mean by this. Are you saying the next line should look like this?

`googleBucket = ((GcsBucket) resource).resolve(true);`

Reply:
yeah i just mean ((GcsBucket) resource).resolve(/*includeUrlprefix=*/true);

Reply:
Done. Thanks!